modules/functional-test/src/main/resources/config.properties

```diff
@@ -39,6 +39,7 @@ api-server-protocol=http
 api-version-string=v1
 user-name=admin
 password=admin
+test-user=wasabi_reader@example.com
```

Comment:
@sudheerkethireddy will there be a test case in the future using wasabi_writer? It will be good to name test-user to test-wasabi-reader. It will be consistent to the userDirectory.properties.